### ISSUE TITLE
How does the 'encode_segmap(mask)' in dataloader.utils work?

### ISSUE THREAD

**Issue by George1ee** [jfzhang95/pytorch-deeplab-xception#95]
I did check the whole codes, but it seems that the "encode_segmap(mask)" in dataloader.utils is not used. Is this true? If so, how the 3D mask image are turned into the 2D label？
Thanks for your nice code again， and looking for response. ：)
